src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/integration/pages/ManageJourneyPricePage.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages
+
+import org.fluentlenium.core.annotation.PageUrl
+
+@PageUrl("http://localhost:8080/search-journeys")
+class ManageJourneyPricePage : ApplicationPage() {
+
+  // Note: this is bypassing the actual search journey page due to not liking the JavaScript in the page where the fields are added by the 3rd party JS lib.
```

Comment:
So at the moment this is a workaround.  Need to look into this some more.  We have something for starters so I am happy with that.